feat: document new option of the container API (#9696)
withastro/docs: src/content/docs/en/reference/container-reference.mdx

});
const json = await response.json();
```

### `partial` option

> **Type**: `boolean`

Reviewer: ```suggestion
<p>

**Type:** `boolean`<br />
**Default:** `true`<br/>
<Since v="4.16.6" />
</p>
```

(I think that will be the next version!)

> By default, the Container API renders components as they were [page partials](/en/basics/astro-pages/#page-partials). This is usually the behavior you want when rendering components.boolean
> 
> However, if you're rendering full-fledged Astro pages, you can opt-out of this behavior by setting `partial` to `false`:

Reviewer: ```suggestion
Whether or not the Container API renders components as if they were [page partials](/en/basics/astro-pages/#page-partials). This is usually the behavior you want when rendering `components.boolean` so you can render components without a full page shell.

To render a component as a full Astro page, including `<!DOCTYPE html>`, you can opt-out of this behavior by setting `partial` to `false`:
```

```js name="Card.test.js"
import Blog from "../src/pages/Blog.astro";

const result = await container.renderToString(Card, {
    partial: false 
});

> console.log(result) // it will contain `<!DOCTYPE html>` at the beginning of the HTML

Reviewer: ```suggestion
console.log(result) // includes `<!DOCTYPE html>` at the beginning of the HTML
```

});
const json = await response.json();
```

### `partial` option

<p>

**Type:** `boolean`<br />
**Default:** `true`<br/>
<Since v="4.16.6" />
</p>

Whether or not the Container API renders components as if they were [page partials](/en/basics/astro-pages/#page-partials). This is usually the behavior you want when rendering `components.boolean` so you can render components without a full page shell.

To render a component as a full Astro page, including `<!DOCTYPE html>`, you can opt-out of this behavior by setting `partial` to `false`:

> ```js name="Card.test.js"

Reviewer 2: ```suggestion
```js name="Card.test.js ins={4}"
```
